apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/src/tour-card.js

```diff
@@ -93,12 +149,59 @@ function CardOverlayControls( { onMinimize, onDismiss } ) {
 					className="welcome-tour-card__minimize-icon"
 					icon={ minimize }
 					iconSize={ 24 }
-					onClick={ () => onMinimize( true ) }
+					onClick={ handleOnMinimize }
 				></Button>
 				<Button isPrimary icon={ close } iconSize={ 24 } onClick={ () => onDismiss() }></Button>
 			</Flex>
 		</div>
 	);
 }
 
+function TourRating() {
+	let isDisabled = false;
+	const tourRating = useSelect( ( select ) => select( 'automattic/nux' ).tourRating() );
+	const { setTourRating } = useDispatch( 'automattic/nux' );
+
+	if ( ! isDisabled && tourRating ) {
+		isDisabled = true;
+	}
+	const rateTour = ( isThumbsUp ) => {
+		if ( isDisabled ) {
+			return;
+		}
+		isDisabled = true;
+		setTourRating( 'thumbs-up' );
```

Comment:
Doesn't this means it's impossible to set `'thumbs-down'` in the data store.

Reply:
> why do we remember the rating in the data store? Yes it allows us to remind the user of the selection they made last time they were in the tour popup, but only for this session.

You are correct to point this out. :) A couple thoughts:

* "only for this session":  As far as I understand it, the value isn't just for this session.  It is set to [persist](https://github.com/Automattic/wp-calypso/blob/aedabce06259cf42585e3779eb25a59434780975/apps%2Fediting-toolkit%2Fediting-toolkit-plugin%2Fwpcom-block-editor-welcome-tour%2Fsrc%2Fstore.js#L59) and I observed the store value persisted after page reload.  Source: [Persistence Plugin](https://github.com/WordPress/gutenberg/blob/a82c257641739fcdec7ed910ad5ddf0cafee9658/packages%2Fdata%2Fsrc%2Fplugins%2Fpersistence%2FREADME.md#L4)
* I debated the best approach for this.  Yes, one implementation could have been to save state in the root component. A very reasonable approach, since right now the goal is just to prevent user from rating multiple times.
* In the future we will (assuming successful A/B test) also show the Tour from the More Menu (same as the current NUX ) and then we might want to know about previous rating, or whether to show the rating at all, or maybe we'd have a different rating question.  

I'd agree that I probably over designed it, and maybe should have simply set up state in a parent component, like `WelcomeTourFrame`, but the above was my thought process.

Reply:
> Doesn't this means it's impossible to set `'thumbs-down'` in the data store.

Ah! That's embarrassing!!!  I think I just hard coded that string for initial dev testing.  Fixed. Thanks for catching!!!

Reply:
> "only for this session": As far as I understand it, the value isn't just for this session. It is set to persist and I observed the store value persisted after page reload.

I missed this 🙇 Since this is persisted in local storage then your implementation looks the best.

I think I was imagining a design in my head where once you've rated the buttons might disappear and get replaced with a "thanks for rating" sort of message, and that felt more of a transient thing. But that's a design change, what you've got here looks good to me now that I understand the persistence thing.